src/braket/aws/aws_quantum_job.py

```diff
@@ -286,6 +286,15 @@ def name(self) -> str:
         """str: The name of the quantum job."""
         return self.metadata(use_cached_value=True).get("jobName")
 
+    @property
+    def _logs_prefix(self) -> str:
```

Comment:
For my information, is it typical to create properties for private functions? Why not just use the private function instead?

Reply:
It's up to you